ai-agents/fhir-questionnaire-generation/data_ingestion_pipeline/pdf_to_md_service/settings.py

```
@@ -25,11 +25,17 @@ class Configs(BaseSettings):
     )
 
+    # FTP or Local File Storage Configs
+    USE_FTP: bool = False  # Set to True to use FTP server, False to use local storage
+
     # FTP Configs
     FTP_HOST: str = "127.0.0.1"
     FTP_PORT: int = 2121
-    FTP_USERNAME: str
-    FTP_PASSWORD: str
+    FTP_USERNAME: str = "ftp_user"
+    FTP_PASSWORD: str = "ftp_password"
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "FTP_PASSWORD"

(S105)